installation/scripts/e2e-testing.sh

```diff
@@ -240,36 +33,95 @@ ADMIN_PASSWORD=$(kubectl get secret admin-user -n kyma-system -o jsonpath="{.dat
 
 helm install "$testcase" --name "${release}" --namespace end-to-end --set global.ingress.domainName="${DOMAIN}" --set-file global.adminEmail=<(echo -n "${ADMIN_EMAIL}") --set-file global.adminPassword=<(echo -n "${ADMIN_PASSWORD}") --tls
 
-helm test "${release}" --timeout 10000 --tls
-testResult=$?
-if [ $testResult -eq 0 ]
-then
-    releasesToClean="$releasesToClean ${release}"
-else
-    exitCode=$testResult
-fi
+suiteName="testsuite-backup-$(date '+%Y-%m-%d-%H-%M')"
+echo "----------------------------"
+echo "- Testing Kyma Backup and Restore functionality..."
+echo "----------------------------"
+
+kc="kubectl $(context_arg)"
 
-checkAndCleanupTest end-to-end
-cleanupResult=$?
-if [ $cleanupResult -ne 0 ]
+${kc} get clustertestsuites.testing.kyma-project.io > /dev/null 2>&1
+if [[ $? -eq 1 ]]
 then
-   exitCode=$cleanupResult
+   echo "ERROR: script requires ClusterTestSuite CRD"
+   exit 1
 fi
 
-for release in $releasesToClean; do
-    cleanupHelmE2ERelease "${release}"
-    cleanupResult=$?
-    if [ $cleanupResult -ne 0 ]
-    then
-        exitCode=$cleanupResult
+matchTests=$(${kc} get testdefinitions --all-namespaces -l 'app=backup-test' -o=go-template='  selectors:
+    matchNames:
+{{- range .items}}
+      - name: {{.metadata.name}}
+        namespace: {{.metadata.namespace}}
+{{- end}}')
+
+cat <<EOF | ${kc} apply -f -
+apiVersion: testing.kyma-project.io/v1alpha1
+kind: ClusterTestSuite
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: ${suiteName}
+spec:
+  maxRetries: 1
+  concurrency: 1
+${matchTests}
+EOF
+
+startTime=$(date +%s)
+
+testExitCode=0
+previousPrintTime=-1
+
+while true
+do
+    currTime=$(date +%s)
+    statusSucceeded=$(${kc} get cts ${suiteName}  -ojsonpath="{.status.conditions[?(@.type=='Succeeded')]}")
```

Comment:
`get cts` and `get clustertestsuites.testing.kyma-project.io` is used in short and long form. Can we stick to one form? :)